docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md

```diff
@@ -24,13 +24,9 @@ weight: 300
 
 You can create and manage recording rules for an external Grafana Mimir or Loki instance. Recording rules calculate frequently needed expressions or computationally expensive expressions in advance and save the result as a new set of time series. Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh.
 
-**Note:**
+> **Note:** Recording rules are executed as instant queries. Evaluations take place at regular intervals according to the group evaluation interval, with a default of 1m. Each group's evaluation interval can be modified, however cannot be less than the min_interval configured within Grafana.
```

Comment:
```suggestion
{{% admonition type="note" %}}
Recording rules are executed as instant queries. Evaluations take place at regular intervals according to the group evaluation interval, with a default of 1m. Each group's evaluation interval can be modified, however cannot be less than the min_interval configured within Grafana.
{{% /admonition %}}
```

Reply:
Consider adding code formatting for min_interval by adding `. I've edited the above suggestion to add the code formatting.